### ISSUE TITLE
Prepare CMS Production environment for JSD transition

### ISSUE THREAD

**Issue by daddison48** [department-of-veterans-affairs/va.gov-cms#3537]
A sequence of manual steps needs to happen before we announce Jira Service Desk to CMS users.

## Acceptance criteria

**Steps to perform after the CMS deployment on Monday, Dec 21 and before the Announcement in #3815**
- [x] "Report a bug" and "Contact Help Desk" page are archived [@rachel-kauff ]
- [x] links to "Report a bug" and "Contact Help Desk" are removed from Support section on CMS help center home page [@rachel-kauff ]
- [ ] CMS help center homepage is updated per recommendation in #3399 + add `target="_blank"` to JSD portal button [@rachel-kauff ]
- [x] Help link in Admin toolbar Manage section is disabled [@oksana-c ]:
<img width="1112" alt="Screen Shot 2020-11-23 at 11 54 55 AM" src="https://user-images.githubusercontent.com/41017988/100008852-d3f55080-2d82-11eb-98e5-da4631ffed4a.png">

- [x] CMS help center second-level menu links are disabled in /admin/structure/menu/manage/documentation [@oksana-c ]
- [x] Help Center CTA block is removed [@oksana-c ]

**Comment by oksana-c** [comment]
many of the ACs here will be dependent on https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3399 

It will be a better experience of going away from email if the CMS has the Helpdesk widget implemented.

/cc @rachel-kauff

**Comment by rachel-kauff** [comment]
@oksana-c Agree! That'll be fun to announce.

**Comment by rachel-kauff** [comment]
Transition comms channels
- Auto response for vacmssupport email addy guiding users to use JSD
- Announcement in #cms-support, CMS, Email to all users
- 3399 places CTA/ access to JSD in the right places 
- ESD autoresponse content (3528 - to do this sprint)
- Socializing change with our own team (3560) - just before

**Comment by erogray** [comment]
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3528 autoresponse probably won't work because it's a distribution group?
Recommending a process instead consisting of:
- canned response for Clarence to use to tell people about the new approach
- then Clarence conveys the body of the request into a Jira ticket
- then Vanessa or Ero reply from Jira

**Comment by rachel-kauff** [comment]
Having trouble getting =target"_blank" to work for the contact help desk button in the rich text ```
field on CMS help landing page. When you get in, @oksana-c I could use help troubleshooting.

`<p><a class="usa-button" href="https://va-gov.atlassian.net/servicedesk/customer/portal/3/group/8/create/26" target="_blank">Contact help desk</a></p>`

^That's what I input in `Source` but when I save it seems to go away.

**Comment by acrollet** [comment]
@rachel-kauff it looks to me like target="_blank" is not allowed in the "Rich Text" text filter. It would be a simple change to add it but there should probably be a separate issue and a spike to decide if we want to do that?

**Comment by oksana-c** [comment]
@acrollet @rachel-kauff target blank is added as a part of the PR that we user for user testing https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3850 
Once that is merged, we'll be able to add target blank attribute on production. 

Since adding target blank is the only remaining AC in this ticket, I'm closing it as complete and will update the button once we're live with the widget.
